client/src/services/firebase.ts

```diff
@@ -0,0 +1,20 @@
+import firebase from "firebase/app";
+//limit import to auth (keep comment to keep linter happy)
+import "firebase/auth";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyApDXStJREUsZymQBGs4yJXi22wc3PBvSo",
```

Comment:
should we store the API key in secret manager instead? just worried about security issues with storing our credentials in the code

Reply:
LOL I THOUGHT THE SAME THING

Reply:
Turns out it should be OK.
https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public

Reply:
I think Firebase API keys aren't actually secrets just because they don't have any security implications - they're just a way for Google to determine which project is which

Reply:
https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public